app/views/partials/attachments_edit_list.scala.html

```diff
@@ -0,0 +1,69 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.request.AuthenticatedRequest
+@import models.Permission
+@import controllers.routes.AttachmentsController
+@import models.StoredAttachment
+@import utils.Dates
+@import views.html.partials.attachment
+@import models.Case
+@(id: String, attachments: Seq[StoredAttachment], showSelect: Boolean = false,
+showRemoval: Boolean = false, caseRef: Option[String] = None, startAtTabIndex : Option[Int] = None)(implicit request: AuthenticatedRequest[_], messages: Messages)
+
+@if(attachments.nonEmpty){
+<legend class="w-100">
+<div id="@{id}-div">
+    <div class="border-bottom display-flex row-alignment" >
+        <div class="bold-small w-75 pl-0">File</div>
+        @if(showSelect) {<div class="bold-small w-25 p-0" id="@{id}-header-select" >Publish</div>}
```

Comment:
based on file name and the logic which is similar in the attachments_list... maybe we dont need showselect param? because it will be always true for this file? I assume... IF not it is ok

Reply:
Thanks Victor. I have removed showselect param as it is no need anymore.